src/components/AnchorList/index.js

```diff
@@ -187,59 +242,125 @@ const AnchorList = ({
-                          ? info.list_image.localFile.childImageSharp.fluid
-                          : ''
-                        return (
-                          <AnchorItemsDes key={index}>
-                            {AnchorImage ? (
-                              <AnimatedImage>
-                                <Img
-                                  fluid={AnchorImage}
-                                  alt="ministries image"
-                                />
-                              </AnimatedImage>
-                            ) : (
-                              ''
-                            )}
-                            <AnchorItemsText>
-                              {info?.list_text ? (
-                                <Text type="body">{info.list_text}</Text>
-                              ) : (
-                                ''
-                              )}
-                            </AnchorItemsText>
-                          </AnchorItemsDes>
-                        )
-                      })
-                    ) : (
-                      <AnchorItemsDes>
-                        {anchorFirstImage ? (
-                          <AnimatedImage>
-                            <Img
-                              fluid={anchorFirstImage}
-                              alt="ministries image"
-                            />
-                          </AnimatedImage>
-                        ) : (
-                          ''
-                        )}
-                        <AnchorItemsText>
-                          {anchorFirstText ? (
-                            <Text type="body">{anchorFirstText}</Text>
-                          ) : (
-                            ''
-                          )}
-                        </AnchorItemsText>
-                      </AnchorItemsDes>
-                    )}
+                    {anchorInfo?.length > 0
+                      ? anchorInfo.map((info, index) => {
+                          const AnchorImage = info?.list_image?.localFile
+                            ?.childImageSharp?.fluid
+                            ? info.list_image.localFile.childImageSharp.fluid
+                            : ''
+                          return (
+                            <Fragment key={` AnchorItemsDes -${index}`}>
+                              <AnchorItemsDes>
+                                {AnchorImage ? (
+                                  <AnimatedImage>
+                                    <Img
+                                      fluid={AnchorImage}
+                                      alt="ministries image"
+                                    />
+                                  </AnimatedImage>
+                                ) : (
+                                  ''
+                                )}
+                                <AnchorItemsText>
+                                  {info?.list_text ? (
```

Comment:
If you are wrapping the text within another text the check should be outside anchoritemstext